examples/specs/area_opacity_xfield.jason

```diff
@@ -0,0 +1,16 @@
+{
+  "data": {"url": "data/unemployment-across-industries.json"},
+  "mark": "area",
+  "encoding": {
+    "x": {
+      "timeUnit": "yearmonth", "field": "date", "type": "temporal",
+      "scale": {"nice": "month"},
+      "axis": {"axisWidth": 0, "format": "%Y", "labelAngle": 0}
+    },
+    "y": {
+      "aggregate": "sum", "field": "count","type": "quantitative"
+    },
+    "opacity": {"field": "date", "type": "temporal"}
```

Comment:
this opacity field is creating weird stacking if you look closely at the output.  

I would say that we better remove this example, since it's not an effectively designed example.